Error when removing signature from database

**Describe the bug**
Removing a signature causes this error from _minhasher_

```sh
2024-03-19 14:53:59,669 [ERROR] rq.worker: [Job 137684eb-06cf-4c59-a627-ad52e65e2d38]: exception raised while executing (minhash_service.tasks.remove_signature)
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/rq/worker.py", line 1428, in perform_job
    rv = job.perform()
         ^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rq/job.py", line 1278, in perform
    self._result = self._execute()
                   ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rq/job.py", line 1315, in _execute
    result = self.func(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/worker/app/minhash_service/tasks.py", line 37, in remove_signature
    status: bool = remove_signature_file(sample_id)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/worker/app/minhash_service/minhash/io.py", line 134, in remove_signature
    tree.remove_many(leaf)
    ^^^^^^^^^^^^^^^^
AttributeError: 'SBT' object has no attribute 'remove_many'
```

**To Reproduce**
Steps to reproduce the behavior:
1. Remove a sample from bonsai gui
2. Check minhasher logs

--- a/.github/workflows/pylint.yml
+++ b/.github/workflows/pylint.yml
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.10"]
+        python-version: ["3.11"]
     steps:
     - uses: actions/checkout@v3
     - name: Set up Python ${{ matrix.python-version }}

--- a/.github/workflows/pytest.yml
+++ b/.github/workflows/pytest.yml
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.10", "3.11", "3.12"]
+        python-version: ["3.11", "3.12"]
     steps:
     - uses: actions/checkout@v3
     - name: Set up Python ${{ matrix.python-version }}

--- a/allele_cluster_service/setup.cfg
+++ b/allele_cluster_service/setup.cfg
@@ -19,8 +19,8 @@ python_requires = >=3.10
 install_requires =
     setuptools
     wheel
-    redis==5.01
-    rq==1.15.1
+    redis
+    rq==1.16.1
     scipy
     pydantic
     numba

--- a/allele_cluster_service/setup.cfg
+++ b/allele_cluster_service/setup.cfg
@@ -19,8 +19,8 @@ python_requires = >=3.10
 install_requires =
     setuptools
     wheel
-    redis
-    rq
+    redis==5.01
+    rq==1.15.1
     scipy
     pydantic
     numba

--- a/api/setup.cfg
+++ b/api/setup.cfg
@@ -29,8 +29,8 @@ install_requires =
     bcrypt==4.1.1
     ldap3==2.9.1
     pandas==2.1.3
-    redis==5.0.1
-    rq==1.15.1
+    redis
+    rq==1.16.1
     motor==3.3.2
     pymongo==4.5.0
     bonsai-prp==0.6.0

--- a/minhash_service/minhash_service/__init__.py
+++ b/minhash_service/minhash_service/__init__.py
@@ -1,3 +1,3 @@
 """Service to perform MinHash based analysis using Sourmash."""
 
-__version__ = "0.1.0"
+__version__ = "0.1.1"

--- a/minhash_service/setup.cfg
+++ b/minhash_service/setup.cfg
@@ -19,11 +19,11 @@ python_requires = >=3.10
 install_requires =
     setuptools
     wheel
-    redis
-    rq
-    sourmash
-    fasteners
-    pydantic
+    redis==5.03
+    rq==1.16.1
+    sourmash==4.8.6
+    fasteners==0.19
+    pydantic==2.6.4
 
 [options.extras_require]
 dev = 

--- a/minhash_service/setup.cfg
+++ b/minhash_service/setup.cfg
@@ -19,8 +19,8 @@ python_requires = >=3.10
 install_requires =
     setuptools
     wheel
-    redis==5.01
-    rq==1.15.1
+    redis
+    rq==1.16.1
     sourmash==4.8.6
     fasteners==0.19
     pydantic==2.5.2

--- a/minhash_service/setup.cfg
+++ b/minhash_service/setup.cfg
@@ -19,11 +19,11 @@ python_requires = >=3.10
 install_requires =
     setuptools
     wheel
-    redis==5.03
-    rq==1.16.1
+    redis==5.01
+    rq==1.15.1
     sourmash==4.8.6
     fasteners==0.19
-    pydantic==2.6.4
+    pydantic==2.5.2
 
 [options.extras_require]
 dev = 

